db/structure.sql

```diff
@@ -61,8 +61,6 @@ CREATE TYPE public.license_types AS ENUM (
 
 SET default_tablespace = '';
 
-SET default_with_oids = false;
```

Comment:
😠  whyyyyyy

Reply:
This shouldn't have been there in the first place. Looks like it was added again in https://github.com/pieforproviders/pieforproviders/pull/319 after being removed in https://github.com/pieforproviders/pieforproviders/pull/313.

I am using Postgres 12.4 and I believe you're as well. @weedySeaDragon Can you check what version of Postgres you're running?